src/middlewares/socialCallback.js

```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line import/no-unresolved
+import Auth from './authenticator';
+import { User } from '../models';
+
+
+const strategyCallback = async (accessToken, refreshToken, profile, done) => {
+  const {
+    id, displayName, photos, emails, username
+  } = profile;
+
+  const user = await User.findOrCreate({
```

Comment:
Let us handle errors in case of failure. A try-catch block should suffice

Reply:
done